pandas/tests/io/test_html.py

```diff
@@ -100,6 +102,7 @@ def test_same_ordering(datapath):
     [
         pytest.param("bs4", marks=td.skip_if_no("bs4")),
```

Comment:
On second thought, html5lib should be added to the marks here and removed as its own thing. (flavors html5lib and bs4 are the same).